front-admin/src/App.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <v-app>
+    <v-app-bar app clipped-left height="60px">
+      <v-app-bar-nav-icon @click.stop="drawer = !drawer"></v-app-bar-nav-icon>
+      <v-toolbar-title> 도란도란 </v-toolbar-title>
+      <v-spacer></v-spacer>
```

Comment:
```suggestion
      <VAppBarNavIcon @click.stop="drawer = !drawer" />
      <v-toolbar-title> 도란도란 </v-toolbar-title>
      <VSpacer />
```
빈 컴포넌트는 셀프 클로징을 하기로 했었어요.